### PR TITLE
Added table driven tests to cephobjectstores_test.go

### PR DESCRIPTION
**Pull request by priyanka19-98**
Signed of by: Priyanka Jiandani <pjiandan@redhat.com>

### CODE REVIEW

**Comment by priyanka19-98** [comment]
@jarrpa Please review

File: pkg/controller/storagecluster/cephobjectstores_test.go [hunk 1]
```diff
@@ -1,21 +1,38 @@
...
 	"github.com/stretchr/testify/assert"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
-	"testing"
 )
 
 func TestCephObjectStores(t *testing.T) {
+	var cases = []struct {
+		label                string
```
Comment (umangachapagain) [inline pkg/controller/storagecluster/cephobjectstores_test.go:17]: wouldn't "name" be a better variable to denote Case name?

File: pkg/controller/storagecluster/cephobjectstores_test.go [hunk 2]
```diff
@@ -1,21 +1,38 @@
...
+		createRuntimeObjects bool
+	}{
+		{
+			label:                "case 1",
```
Comment (umangachapagain) [inline pkg/controller/storagecluster/cephobjectstores_test.go:21]: Try to be more specific. What's this test case about?

**Review by umangachapagain — CHANGES_REQUESTED** [review]

**Review by umangachapagain — CHANGES_REQUESTED** [review]
PR and Commit title is too generic. Try to be more specific.

File: pkg/controller/storagecluster/cephobjectstores_test.go (hunk 1 above)
Comment (priyanka19-98) [reply pkg/controller/storagecluster/cephobjectstores_test.go:17]: Every test file uses the `label` variable to denote case name. I have added a comment beside the case. Let me know if other changes are required.

**Review by jarrpa — CHANGES_REQUESTED** [review]
Looks good! The commit message still needs  to be updated to match the PR subject.

**Comment by priyanka19-98** [comment]
> Looks good! The commit message still needs to be updated to match the PR subject.

Done!!

**Review by jarrpa — APPROVED** [review]

**Pull request merged**